DEBUG 2016-05-24 13:49:55,181: 	 Executing command: 'submit'
DEBUG 2016-05-24 13:49:55,182: 	 Found CRAB cache file /uscms/home/sdurgut/.crab3
DEBUG 2016-11-28 11:34:45,556: 	 Executing command: 'status'
DEBUG 2016-11-28 11:34:45,560: 	 Found CRAB cache file /uscms/home/sdurgut/.crab3
DEBUG 2016-11-28 11:34:45,606: 	 Checking credentials
DEBUG 2016-11-28 11:34:45,606: 	 Found proxy file /tmp/x509up_u46497
DEBUG 2016-11-28 11:34:45,606: 	 Getting proxy life time left
DEBUG 2016-11-28 11:34:45,709: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u46497 -timeleft
 output : 0

 error: 
 retcode : 0
DEBUG 2016-11-28 11:34:45,709: 	 Proxy valid for 00:00:00 hours
DEBUG 2016-11-28 11:34:45,709: 	 Creating new proxy for 24:00 hours with VO group '' and VO role 'NULL'.
ERROR 2016-11-28 11:54:45,725: 	 Timeout in voms-proxy-init -voms cms:/cms -valid 24:00 -rfc execution.
DEBUG 2016-11-28 11:54:45,978: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u46497 -timeleft
 output : 0

 error: 
 retcode : 0
DEBUG 2016-11-28 11:54:46,081: 	 Executing : 
 command : env X509_USER_PROXY=/tmp/x509up_u46497 voms-proxy-info -fqan
 output : /cms/Role=NULL/Capability=NULL

 error: 
 retcode : 0
ERROR 2016-11-28 11:54:46,082: 	 Problems creating proxy.
ERROR 2016-11-28 11:54:46,082: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 292, in __init__
    self.proxyCreated = self.proxy.createNewVomsProxySimple(timeLeftThreshold = 720)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1611-comp/lib/python2.7/site-packages/CRABClient/CredentialInteractions.py", line 152, in createNewVomsProxySimple
    raise ProxyCreationException("Problems creating proxy.")
ProxyCreationException: Problems creating proxy.
